contracts/migrations/4_user_replica_set_migration.js

```diff
@@ -19,7 +19,7 @@ module.exports = (deployer, network, accounts) => {
     const config = contractConfig[network]
     let registry
     let registryAddress
-    if (network === 'test_local' || network === 'development') {
+    if (network === 'test_local' || network === 'development' || network === 'audius_dev') {
```

Comment:
If you're changing the network, double check that that contract-config was the registryAddress written to, or else the user replica set contract migration will not run because it can't find the registry

Reply:
This PR tries predetermine all addresses, so we don't require the contract config.